topcoffea/params/xsec.yml

```diff
@@ -209,4 +209,47 @@ GluGlutoContinto2Zto4TauK_13p6TeV       : 0.0051
 TbarWplustoLNu2Q_13p6TeV           : 19.36
 TWminustoLNu2Q_13p6TeV             : 19.36
 
+#NEW from multileptons for Run3
```

Comment:
We will need a comment with references for these (like a link, or explanation pointing to where they came from)

Reply:
Looks like this comment is still open. Do you have references for these?

Reply:
Done with the forthcoming commit